VideoWeb/VideoWeb/ClientApp/src/app/services/connection-status.service.ts

```diff
@@ -75,7 +76,10 @@ export class ConnectionStatusService {
 
         if (this.status === connectionResult) {
             this.logger.info(`${this.loggerPrefix} ${this.status ? 'Online' : 'Offline'}`);
-            this.connectionStatus.next(this.status);
+            const currentPathName = this.locationService.getCurrentPathName();
+            if (currentPathName.indexOf('waiting-room') < 0) {
```

Comment:
Hi
any reason we're doing the check in the connection service and not the subscription side?

Reply:
The connection status services responsibility is to report the status change
The subscriber's responsibility is to redirect on error (which we want to stop)
If we change in the service, any other future uses may be negatively affected by this change?

While i think this will solve the current problem, im not sure its correct?